colabfold/alphafold/models.py

```diff
@@ -16,12 +17,26 @@ def load_models_and_params(
     recompile_all_models: bool = False,
     stop_at_score: float = 100,
     rank_by: str = "plddt",
+    return_representations: bool = False
 ) -> List[Tuple[str, model.RunModel, haiku.Params]]:
     """We use only two actual models and swap the parameters to avoid recompiling.
 
     Note that models 1 and 2 have a different number of parameters compared to models 3, 4 and 5,
     so we load model 1 and model 3.
     """
+
+    if return_representations:
+        # this forces the AlphaFold to always return representations
+        # by means of a decorator on the __call__
+
+        def force_return_representation(fn):
+            @wraps(fn)
+            def inner(*args, **kwargs):
+                return fn(*args, return_representations=True, **kwargs)
```

Comment:
You can reduce this to two lines with [functools.partial](https://docs.python.org/3/library/functools.html#functools.partial)

Reply:
:pray:

Reply:
found out that it should be https://docs.python.org/3/library/functools.html#functools.partialmethod instead